hadoop-hdds/server-scm/src/test/java/org/apache/hadoop/hdds/scm/ha/TestSCMHAConfiguration.java

```diff
@@ -60,26 +60,17 @@ public void setup() {
 
   @Test
   public void testSetStorageDir() {
-    SCMHAConfiguration scmhaConfiguration = conf.getObject(
-        SCMHAConfiguration.class);
-    scmhaConfiguration.setRatisStorageDir("scm-ratis");
-    conf.setFromObject(scmhaConfiguration);
-
-    scmhaConfiguration = conf.getObject(
-        SCMHAConfiguration.class);
-    Assert.assertEquals("scm-ratis", scmhaConfiguration.getRatisStorageDir());
+    conf.set(ScmConfigKeys.RATIS_STORAGE_DIR, "scm-ratis");
+    Assert.assertEquals("scm-ratis", conf.get(ScmConfigKeys.RATIS_STORAGE_DIR));
```

Comment:
This test seems unnecessary now, since we removed the  _SCMHAConfiguration_ class.